convert2rhel/unit_tests/actions/pre_ponr_changes/backup_system_test.py

```diff
@@ -342,17 +357,27 @@ def test_backup_package_file_complete(
                 # Check if the file exists and contains right content
                 with open(backed_up_file_path, mode="r") as f:
                     assert f.read() == "Content for testing of file %s" % original_file_path
-                os.remove(original_file_path)
+                # Remove the original file
+                try:
+                    os.remove(original_file_path)
+                    removed_paths.append(original_file_path)
+                # FileNotFound on Python 3+, due compatibility with Python 2.7 using OSError
+                except OSError:
+                    # If the path is present multiple times in the 'rpm -Va' output
+                    # it's possible, the path was already removed. If not, that's fail.
+                    if original_file_path not in removed_paths:
+                        assert False
```

Comment:
@Venefilyn I checked this and the comments `###` to have this prepared. So both of the commits are ready and we can discuss which we will merge.

Comments are updated, but I don't think this can be changed. I'm also checking if the path was really removed and the exception doesn' have to be raised each time.